androidgen/src/main/java/com/azure/autorest/android/template/AndroidEnumTemplate.java

```diff
@@ -29,7 +30,10 @@ protected void addGeneratedImport(Set<String> imports) {
     }
 
     @Override
-    protected void addGeneratedAnnotation(JavaContext classBlock) {
+    protected void addGeneratedAnnotationForExpandableStringEnum(JavaContext context) {
     }
 
+    @Override
+    protected void addGeneratedAnnotationForEnum(JavaEnum context) {
+    }
```

Comment:
nit, probably OK with same name as overload `addGeneratedAnnotation`.

Reply:
```java
protected void addGeneratedAnnotationForExpandableStringEnum(JavaContext context) {
    }


  protected void addGeneratedAnnotationForEnum(JavaEnum context) {
  }
```
The two methods takes different input parameter type, so I add the suffix  to make it clearer. Do you mean we call the two methods `addGeneratedAnnotationForExpandableStringEnum` and `addGeneratedAnnotation`?

Reply:
I mean we probably call both `addGeneratedAnnotation`. (just different parameter)

Ideally I feel `JavaEnum` should also inherit from `JavaContext`, so it would be a single method. But no need to do it here.